### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by chrissimpkins** [sf-playground/autofont#13]
My current understanding of Windows support for this tool is:

- developers on Windows would need to have Cygwin or MinGW + autotools installed
- users on Windows would need to have Cygwin or MinGW without autotools installed

Is this correct?

**Comment by chrissimpkins** [comment]
cc @texhex 

Thoughts? Configurable typeface build tool built on autotools.

**Comment by puppetjoy** [comment]
Correct.  This will also be supported on the Linux subsystem for Windows.

As a Cygwin user myself, I'll be targeting that most closely because I think it presents the easiest option for users and developers alike.  It has a simple GUI installer that takes just a couple of clicks to select git, make, automake, autoconf, python, gcc, etc. packages.  Couple clicks later and you're in a shell where you can '`git clone; ./configure; make; etc.`'.

**Comment by chrissimpkins** [comment]
> Couple clicks later and you're in a shell where you can 'git clone; ./configure; make; etc.'.

👍
